docs/data/data-strategy.md

```diff
@@ -0,0 +1,87 @@
+# Data Strategy
+
+Mission: Help Thoth Tech Make Better Decisions, Faster.
+
+At Thoth Tech, we endeavor to be user-centered. Users will be the center of our operations including
+how we develop and support our products. To support that mission, we would like to have strong data
+insights.
+
+We are creating a Data Platform, a single unified data and analytics stack. The Data Platform will
+power cross-functional reporting and analysis, and in general, allow all team members to make better
+decisions with trusted data. Over time, the Data Platform will further enhance Thoth Tech's
+analytics capabilities with features such as data products - enriched and aggregated data integrated
+into Thoth Tech products for use by our users. This acceleration happens through the development of
+"Data Flywheels".
+
+<details>
```

Comment:
This is a cool feature

Reply:
Indeed. It helps to reduce attention span to the main content but still leaves room for the curious. I will add this to the markdown guide.

Reply:
Raised an issue to follow up https://github.com/thoth-tech/handbook/issues/65